DeusEx/Classes/DXREntranceRando.uc

```diff
@@ -647,11 +711,11 @@ function EntranceRando(int missionNum)
             RandoMission12();
             break;
         case 14:
-            RandoMission14();
+            //RandoMission14();
             break;
         case 15:
             RandoMission15();
```

Comment:
area51 has some annoying 1-ways, you can end up unable to get back to 15_AREA51_FINAL, but there is at least 1 ending that you can complete just inside of 15_Area51_page